underwriter/Dockerfile

```
@@ -244,7 +244,19 @@ RUN groupadd --gid 3434 circleci \
 RUN sudo pip install pipenv
 # END IMAGE CUSTOMIZATIONS
 
-# Install thrift
-RUN apt-get install -y thrift-compiler
+# Install thrift 0.13.0
+RUN buildDeps='automake bison flex libtool g++ libboost-dev libboost-filesystem-dev libboost-program-options-dev libboost-system-dev libboost-test-dev libssl-dev libevent-dev make pkg-config' \
+    && set -ex \
+    && apt-get update && apt-get install -y --no-install-recommends $buildDeps \
+    && rm -rf /var/lib/apt/lists/* \
+    && mkdir /thrift-build/ && cd /thrift-build/ \
+    && curl -fsSL --compressed "https://codeload.github.com/apache/thrift/tar.gz/v0.13.0" -o thrift.tgz \
+    && tar xfz thrift.tgz \
+    && cd thrift-0.13.0 \
+    && ./bootstrap.sh \
+    && ./configure --prefix=/usr/local/ --without-as3 --without-cpp --without-qt5 --without-c_glib --without-csharp --without-java  --without-erlang --without-nodejs --without-nodets --without-lua --without-python --without-perl --without-php --without-php_extension  --without-dart --without-ruby --without-haskell --without-go --without-swift --without-rs --without-cl --without-haxe --without-dotnetcore --without-d \
```

Do we definitely want `--without-python`? It seems like that would disable the ability to build Python stubs.

Also you might be able to write it like this:
```suggestion
    && ./configure --prefix=/usr/local/ --enable-libs=no --with-python=yes \
```

---

Ohhh, I see, `python` is python 2. `py3` is python 3. So this should be good as-is. You might still be able to change it if you wanted to shorten it. But you'd have to test this, of course:

```suggestion
    && ./configure --prefix=/usr/local/ --enable-libs=no --with-py3=yes
```

---

I'll try `--enable-libs=no`. We are only trying to build the compiler here. We source the lib directly from the pip package manager for the projects using it.

---

Ah, I was assuming that the `--with-py3` meant it would support generating Python 3 stubs n' stuff. But I think you're right--I think "libs" refer to the run-time libraries for each language.